lnwallet/chanfunding/canned_assembler.go

```diff
@@ -131,12 +142,16 @@ type CannedAssembler struct {
 
 	// initiator indicates if we're the initiator or the channel or not.
 	initiator bool
+
+	// frozen indicates if the channel to be created is intended to be
+	// frozen.
+	frozen bool
 }
 
 // NewCannedAssembler creates a new CannedAssembler from the material required
 // to construct a funding output and channel point.
-func NewCannedAssembler(chanPoint wire.OutPoint, fundingAmt btcutil.Amount,
-	localKey *keychain.KeyDescriptor,
+func NewCannedAssembler(frozen bool, chanPoint wire.OutPoint,
```

Comment:
This commit won't compile without changing the callers too. Could just add a static false everywhere and then fix in the later commits.